src/components/atoms/SpacesDropdown/SpacesDropdown.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useMemo, useState } from "react";
+import { Dropdown as ReactBootstrapDropdown } from "react-bootstrap";
```

Comment:
I like this change. It just doesn't appear all of the `<Dropdown.Item ... ` have been renamed to `<ReactBootstrapDropdown.Item ... `

Reply:
fixed in 919b1f031c1e636c9de7dbd1aabb12ce7f337fef